Dockerfile

```diff
@@ -9,8 +9,7 @@ RUN apt-get install -y wget curl git xvfb build-essential libavahi-compat-libdns
 RUN apt-get install -y python python-dev python-pip python-virtualenv
 
 # Nodejs
-RUN curl -sL https://deb.nodesource.com/setup_0.12 | bash -
-RUN apt-get install -y nodejs
+RUN curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.31.1/install.sh | bash
```

Comment:
yes, we will provide a .nvmrc

this will allow us to easily update the version of node without having to rebuild the image